opentech/apply/funds/templates/funds/includes/status_bar.html

```diff
@@ -1,17 +1,26 @@
 <div class="status-bar">
-    {# '.status-bar__item--is-complete' needs to be added to each 'complete' step #}
-    {# '.status-bar__item--is-current' needs to be added to the current step #}
-    {% for phase in workflow %}
-        <div class="status-bar__item {% if phase == status %}status-bar__item--is-current{% endif %}">
-            <span class="status-bar__tooltip" data-title="{{ phase.name }}" aria-label="{{ phase.name }}"></span>
+    {% for phase in status.stage %}
+        <div class="status-bar__item
+                    {% if phase == status %}
+                        status-bar__item--is-current
+                    {% elif phase.step < status.step %}
+                        status-bar__item--is-complete
+                    {% endif %}">
+            <span class="status-bar__tooltip"
+                {% if phase == status %}
+                    data-title="{{ status.name }}" aria-label="{{ status.name }}"
+                {% else %}
+                    data-title="{{ phase.name }}" aria-label="{{ phase.name }}"
+                {% endif %}
```

Comment:
this if feels redundant. when `phase == status` one can simply output `phase.name` as it doesn't make a difference.

Reply:
well... 

the follow proves equality (`phase==status`)
https://github.com/OpenTechFund/opentech.fund/pull/128/files#diff-75e081f2231c82e6d73565e9d6bca75aR192 

But this then displays the name as "outcome"
https://github.com/OpenTechFund/opentech.fund/pull/128/files#diff-75e081f2231c82e6d73565e9d6bca75aR187

so unless you use the actual phase rather than the iterated phase you never get the actual phase name. hence the switcheroo.

Reply:
make a note of it ;)